README.md

```diff
@@ -100,7 +120,9 @@ Given a document like
 </main>
 ```
 
-If we called `document.getElementById('app').querySelectorAll('ul.ranked-list li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>, <li>11</li>`.
+If we called `document.getElementById('app').querySelectorAll('ul.ranked-list
+li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>,
+<li>11</li>`.
 
 We could change the `.innerHTML` of these `li`s like so:
```

Comment:
I'm not sure where innerHTML is introduced, but is there a compelling reason to not swap all uses of it with _content_ versus innerHTML?